README.rst

```diff
@@ -177,6 +177,27 @@ currently the compiler on these Macintosh Personal Computers do not support open
 The compiler is also gives extra warnings, and remove `-Werror` from the Makefile.local
 is recommended.
 
+Docker
+------
+
+There is a basic docker configuration file to set up a container
+for FastPM. 
+
+To build it, run:
+
+.. code::
+
+    docker build -t fastpm .
+
+To start the docker container in interactive mode,
+with port 8888 exposed and with
+``/my/file/directory`` linked, run
```

Comment:
These could be added to the Dockerfile, if you would prefer. Or the commands could be run manually inside the docker container once it is started.
With `pip`, the docker commands would be (could be added after line 24, or the `apt-get` could be merged with the one on line 8):

````
RUN apt-get update && apt-get install -qqy \
    python \
    python-dev

RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
    python get-pip.py && \
    rm get-pip.py

RUN pip --no-cache-dir install \
    setuptools \
    numpy \
    jupyter
````

Then, once jupyter is installed, the command would be `jupyter notebook --ip=* --allow-root`, and you could open a browser at localhost:8888.